azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/localuser/DragTouchListener.kt

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.presentation.fragment.calling.localuser
+
+import android.view.MotionEvent
+import android.view.View
+
+class DragTouchListener : View.OnTouchListener {
+
+    private var curX = 0f
+    private var curY = 0f
+    private var startX = 0f
+    private var startY = 0f
+    private var tranX = 0f
+    private var tranY = 0f
+    private lateinit var view: View
+    private var listener: OnDragListener? = null
```

Comment:
Where is this being set?

Reply:
thx for pointing this, I cleaned up the unused listener now.